### PR TITLE
Maltiverse Connector Actions an Enrichment Playbooks

### PR DESCRIPTION
**Pull request by cs-pratap-nikam**
#### PMDB
https://pmdb.fortinet.com/ProjectManagement/project/detail/26452

#### Descriptions:
Maltiverse Connector Actions an Sample Plabooks and Enrichment Playbooks

UTCs:
- [x] Connector installation verified.
- [x]  Connector logo verified.
- [x]  Actions and Playbook list verified.
- [x]  Playbook tags verified.
- [x]  All Playbook are in info mode verified.
- [x]  All Playbook are in inactive mode verified.
- [x]  Check health verified.
- [x]  Verified all actions.

### CODE REVIEW

File: maltiverse/info.json [hunk 1]
```diff
@@ -0,0 +1,264 @@
+{
+    "name": "maltiverse",
+    "label": "Maltiverse",
+    "version": "1.0.0",
+    "description": "Maltiverse Threat Intelligence Feeds can be integrated with your security stack to provide improvement in terms of detections and protection capabilities from different points of view. You can also upload an deploy your own Threat Intelligence!",
+    "publisher": "Fortinet",
+    "cs_approved": false,
```
Comment (cs-omkar-kumbhar) [inline maltiverse/info.json]: I think this connector will be certified, so you can mark this as true

File: maltiverse/operations.py [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+logger = get_logger('maltiverse')
+
+
+class Maltiverse(object):
+
+    def __init__(self, config):
+        self.server_url = config.get('server_url').strip()
+        self.api_key = config.get('api_key')
+        self.headers = {'accept': 'application/json', 'Authorization': 'Bearer {}'.format(self.api_key)}
+        if not self.server_url.startswith('https://'):
```
Comment (cs-omkar-kumbhar) [inline maltiverse/operations.py]: please change the if condition to this
if not self.server_url.startswith('https://') and not self.server_url.startswith('http://'):

File: maltiverse/info.json (hunk 1 above)
Comment (cs-pratap-nikam) [reply maltiverse/info.json]: marked it as true

File: maltiverse/operations.py (hunk 2 above)
Comment (cs-pratap-nikam) [reply maltiverse/operations.py]: added the required condition

**Review by cs-omkar-kumbhar — APPROVED** [review]

**Review by cs-prerna-joshi — APPROVED** [review]

**Pull request merged**